ovs/lib/disk.py

```diff
@@ -71,7 +71,7 @@ def sync_with_reality(storagerouter_guid=None):
                 context = remote.Context()
                 devices = [device for device in context.list_devices(subsystem='block')
                            if ('ID_TYPE' in device and device['ID_TYPE'] == 'disk') or
-                              ('DEVNAME' in device and ('loop' in device['DEVNAME'] or 'nvme' in device['DEVNAME']))]
+                              ('DEVNAME' in device and ('loop' in device['DEVNAME'] or 'nvme' in device['DEVNAME'] or 'md' in device['DEVNAME']))]
```

Comment:
I'd change this to 'startswith' instead of 'md' in ...
Same for loop and nvme as far as i'm concerned (slightly safer)